client/eslint.config.js

```diff
@@ -0,0 +1,39 @@
+import prettier from "eslint-config-prettier";
+import js from "@eslint/js";
+import { includeIgnoreFile } from "@eslint/compat";
+import svelte from "eslint-plugin-svelte";
+import globals from "globals";
+import { fileURLToPath } from "node:url";
+import svelteConfig from "./svelte.config.js";
+const gitignorePath = fileURLToPath(new URL("./.gitignore", import.meta.url));
+
+/** @type {import('eslint').Linter.Config[]} */
+export default [
+	includeIgnoreFile(gitignorePath),
+	js.configs.recommended,
+	...svelte.configs["flat/recommended"],
+	prettier,
+	...svelte.configs["flat/recommended"],
+	{
+		languageOptions: {
```

Comment:
Do we not need to specify `ecmaVersion` any more?

Reply:
No, that's fine then! I think I had it in mostly so that if "latest" gets bumped, it doesn't immediately break our CI, but it's probably fine.